RELEASE.md

```diff
@@ -2,49 +2,61 @@
-| v0.56   | 2022-04-20                                 | Simon Pasquier (GitHub: @simonpasquier) |
-| v0.57   | 2022-06-01                                 | Simon Pasquier (GitHub: @simonpasquier) |
-| v0.58   | 2022-07-13                                 | Pawel Krupa (GitHub: @paulfantom)       |
-| v0.59   | 2022-08-24                                 | Philip Gough (GitHub: @PhilipGough)     |
-| v0.60   | 2022-10-05                                 | Simon Pasquier (GitHub: @simonpasquier) |
-| v0.61   | 2022-11-16                                 | Filip Petkovski (Github: @fpetkovski)   |
-| v0.62   | 2022-12-28                                 | Simon Pasquier (GitHub: @simonpasquier) |
-| v0.63   | 2023-02-08                                 | Philip Gough (GitHub: @PhilipGough)     |
-| v0.64   | 2023-03-22                                 | Simon Pasquier (GitHub: @simonpasquier) |
-| v0.65   | 2023-05-03                                 | Philip Gough (GitHub: @PhilipGough)     |
-| v0.66   | 2023-06-14                                 | Arthur Sens (Github: @ArthurSens)       |
-| v0.67   | 2023-07-26                                 | Simon Pasquier (GitHub: @simonpasquier) |
-| v0.68   | 2023-09-06                                 | Arthur Sens (Github: @ArthurSens)       |
-| v0.69   | 2023-10-18                                 | Simon Pasquier (GitHub: @simonpasquier) |
-| v0.70   | 2023-11-29                                 | Pawel Krupa (GitHub: @paulfantom)       |
-| v0.71   | 2024-01-10                                 | Simon Pasquier (GitHub: @simonpasquier) |
-| v0.72   | 2024-02-21                                 | Arthur Sens (Github: @ArthurSens)       |
-| v0.73   | 2024-04-03                                 | Jayapriya Pai (Github: @slashpai)       |
-| v0.74   | 2024-05-15                                 | **searching for volunteer**             |
 | v0.75   | 2024-06-26                                 | **searching for volunteer**             |
+| v0.74   | 2024-05-15                                 | **searching for volunteer**             |
+| v0.73   | 2024-04-03                                 | Jayapriya Pai (Github: @slashpai)       |
+| v0.72   | 2024-02-21                                 | Arthur Sens (Github: @ArthurSens)       |
+| v0.71   | 2024-01-10                                 | Simon Pasquier (GitHub: @simonpasquier) |
+| v0.70   | 2023-11-29                                 | Pawel Krupa (GitHub: @paulfantom)       |
+| v0.69   | 2023-10-18                                 | Simon Pasquier (GitHub: @simonpasquier) |
+| v0.68   | 2023-09-06                                 | Arthur Sens (Github: @ArthurSens)       |
+| v0.67   | 2023-07-26                                 | Simon Pasquier (GitHub: @simonpasquier) |
+| v0.66   | 2023-06-14                                 | Arthur Sens (Github: @ArthurSens)       |
+| v0.65   | 2023-05-03                                 | Philip Gough (GitHub: @PhilipGough)     |
```

Comment:
I would also remove all entries from there. From my POV, the idea of keeping the history is to have a rough idea of who did the release in the past and have a healthy rotation.

Reply:
done